### PR TITLE
Zac suggested edits

### PR DESCRIPTION
**Pull request by zklamm**
These are suggested edits for the Introduction and Serverless sections of the case study.

Several of the changes are just adding captions or a figure, so we'll want to include those.
Others are just minor edits.

The only change of substance is the suggestion to remove the paragraph about cold starts and how AWS offers a service to help with that. I feel that it isn't particularly relevant to our discussion but could be convinced otherwise.

### CODE REVIEW

**Review by T-monius — CHANGES_REQUESTED** [review]

File: index.html [hunk 1]
```diff
@@ -190,24 +189,11 @@ <h3>2.3 AWS Lambda</h3>
...
-          historically taken hundreds of milliseconds for a container hadn't
-          already been operating. Hence the notion of a 'cold start' meaning
-          that a first invocation of a given Lambda required a period of
-          extended latency. AWS has recently offered a feature to address 'cold
-          start' through sub 100 millisecond spin up of certain function
-          containers. The drawback of this new feature is an added cost, so it
-          doesn't seem to be a great fit for use with all Lambdas ins a workflow
-          since users of AWS Lambda are already charged for execution time of
-          Lambda in 100 ms increments.
-        </p>
```
[the remark this answers is not in the document]
Comment (zklamm) [reply index.html:204]: Works for me. Let's keep it in.

[the remark this answers is not in the document]
Comment (johnisom) [reply index.html:204]: Second this. We talk about cold starts elsewhere so we need to at least say what it is

File: assets/css/main.css [hunk 2]
```diff
@@ -356,6 +356,15 @@ h3 {
   text-align: center;
 }
 
+#serverless-patterns {
+  font-weight: 300;
+}
+
+#orchestration {
```
Comment (johnisom) [inline assets/css/main.css:363]: This is problematic, because there is already an element with this id (heading of section 3)

File: index.html [hunk 3]
```diff
@@ -228,10 +214,17 @@ <h3>2.4 Serverless Patterns</h3>
...
-          Authorization 5) Orchestration.
+          five categories:
         </p>
 
+        <ul id="serverless-patterns">
+          <li>Availability</li>
+          <li>Event Management</li>
+          <li>Communication</li>
+          <li>Authorization</li>
+          <li id="orchestration">Orchestration</li>
```
Comment (johnisom) [inline index.html:225]: Related to my comment in the CSS

**Review by johnisom — CHANGES_REQUESTED** [review]

File: assets/css/main.css (hunk 2 above)
Comment (zklamm) [reply assets/css/main.css:363]: Ooh, good catch.

File: index.html [hunk 4]
```diff
@@ -323,23 +316,9 @@ <h3>3.1 Orchestration Patterns</h3>
             alt="Initial components in workflow derived from task"
             title="Thin workflow"
           />
-          <figcaption>Figure 2:</figcaption>
+          <figcaption>Figure 2: Example Workflow</figcaption>
         </figure>
 
-        <ol>
```
Comment (johnisom) [inline index.html:329]: Good idea on axing this. It was ugly and didn't matter

File: assets/css/main.css [hunk 5]
```diff
@@ -123,7 +123,7 @@ img {
 }*/
 
 #case-study img {
-  max-width: 50vw;
+  max-width: 33vw;
```
Comment (johnisom) [inline assets/css/main.css:126]: This is good. However it is small on mobile devices. Could we include a media query to set the max width value so the pictures are wider on mobile devices but smaller on large screens?

**Review by johnisom — CHANGES_REQUESTED** [review]

File: assets/css/main.css (hunk 5 above)
Comment (zklamm) [reply assets/css/main.css:126]: Yes, I'll do that.

File: index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (zklamm) [reply index.html:204]: OK, I committed a change that adds this back in. There were a few typos that were corrected as well.

File: assets/css/main.css (hunk 2 above)
Comment (zklamm) [reply assets/css/main.css]: Correction has been committed.

File: index.html (hunk 3 above)
Comment (zklamm) [reply index.html]: Correction has been committed.

File: assets/css/main.css (hunk 5 above)
Comment (zklamm) [reply assets/css/main.css]: So I don't this change made the images smaller in the mobile view, but I agree that the images do need to be larger when in the mobile view. So I just committed a change that updates this property for all the different media queries. As you update the size of your window the responsive design should yield decently sized diagrams.

**Review by T-monius — APPROVED** [review]
Looks good to me))

**Review by johnisom — APPROVED** [review]

**Pull request merged**